### DISCUSSION TITLE
Echarts - lib de Charts do Baidu

### DISCUSSION THREAD

**Discussion by jonatas-c-pereira** [frontendbr/forum#1172] [Perguntas e Respostas]
E aí galera.

Queria discutir com vocês sobre a biblioteca de charts do Baidu que estou usando nos meus projetos.
É muito difícil achar conversas sobre ele na interwebs, contudo a documentação dele é bem completa, o que deixa bem suave de usar.
Achei interessante compartilhar, ele usa WebGL para algumas coisas, licença Apache v2 e não pesa tanto quanto o d3.
Alguém  #conhece?
[http://echarts.baidu.com/](url)
[https://github.com/apache/incubator-echarts](url)

**Comment by simaodeveloper** [comment]
Não conhecia, muito legal, vou ler mais sobre na documentação,

Seguem os links corrigidos:

[http://echarts.baidu.com/](http://echarts.baidu.com/)
[https://github.com/apache/incubator-echarts](https://github.com/apache/incubator-echarts)

**Comment by luizbills** [comment]
Também não conhecia, vou dar uma olhada depois.

Mas bate um medo de aparecer janelas pedindo para eu instalar outros softwares :laughing:
